### ISSUE TITLE
Clear Filter needs Orchid specificity for "Clear ____ Filter"

### ISSUE THREAD

**Issue by erinchambers** [CDRH/family_letters#199]
With the updates and pulling in the recent orchid changes, the specific category no longer displays when you try to clear a filter (e.g. "Clear Category Filter" text is now "Clear Filter" text when you hover over the X).

We'll want to build in some multilingual support for this. Instead of our `search.clear` variable in `*.yml` being "Clear Filter," we should have it broken up into "Clear" and "Filter", two separate variables (maybe `search.clear` and `search.filter`).

**Comment by erinchambers** [comment]
By all rights, this should be remediating by switching Family Letters to the Orchid branch that's already been remediated, because Family Letters doesn't override `summary_boxes.html.erb`... but it's not showing up on the dev site. I think #204 is probably the culprit.

**Comment by erinchambers** [comment]
Got it!
